### DISCUSSION TITLE
How to connect HA to docker instance of MA 2.0 server?

### DISCUSSION THREAD

**Discussion by joshlawless** [music-assistant/support#1175] [Q&A] [answered]
I have MA 2.0 running in its own Docker container, everything seems like it works (I'm new to MA), but what do I do in my HA instance (also running in a Docker container) so that it can interact with MA?  Sorry for the elementary question, but I can't find the answer anywhere.

**Comment by joshlawless** [comment]
The readme at https://github.com/music-assistant/frontend explains the development environment, but not how to get what I assume, based on the name, is the right component, up and running in HA

**Comment by OzGav** [comment, marked as answer]
The integration to connect to HA is not yet written. Will be a couple of weeks.

**Comment by joshlawless** [comment]
More time to get my new Soundavo setup up and running.  :-D
